[gui] Rename review status and detection status texts (#2908)
Ericsson/codechecker: docs/web/user_guide.md

  --report-hash [REPORT_HASH [REPORT_HASH ...]]
                        Filter results by report hashes.
  --review-status [REVIEW_STATUS [REVIEW_STATUS ...]]
                        Filter results by review statuses.
                        Reports can be assigned a review status of the
                        following values:

>                         - Unreviewed (default): Nobody has seen this report.

Reviewer: I think the `default` here and the `default` filter value (which contains the confirmed value too) a few lines below can be a bit confusing.
How about removing the default word from here or to extend the description a bit? (`default value for each report` or something like this)

                        - Confirmed: This is really a bug.
                        - False positive: This is not a bug.
                        - Intentional: This report is a bug but we don't want
                        to fix it. (default: ['unreviewed', 'confirmed'])
  --detection-status [DETECTION_STATUS [DETECTION_STATUS ...]]
                        Filter results by detection statuses.
                        The detection status is the latest state of a bug

>                         report in a run. When a report id is first detected it

Reviewer 2: ```suggestion
                        report in a run. When a unique report is first detected it
```

>                         will be marked as New. When the reports stored again

Reviewer 2: ```suggestion
                        will be marked as New. When the report is stored again
```

>                         with the same run name then the detection status can
>                         change to one of the following options:

Reviewer 2: ```suggestion
                        with the same run name then the detection status
                        changes to one of the following options:
```

                        - Resolved: when the bug report can't be found after
                        the subsequent storage.
                        - Unresolved: when the bug report is still among the
                        results after the subsequent storage.

>                         - Reopened: when a resolved bug appears again.

Reviewer 2: ```suggestion
                        - Reopened: when a Resolved bug appears again.
```

>                         - Off: were reported by a checker that is switched off

Reviewer 2: ```suggestion
                        - Off: The bug was reported by a checker that was switched off
```

                        during the last analysis which results were stored.

>                         - Unavailable: were reported by a checker that does
>                         not exists anymore because it was removed or renamed.

Reviewer 2: ```suggestion
                        - Unavailable: were reported by a checker that does
                        not exists in the analyzer anymore because it was removed or renamed.
```
